src/api/api.js

```diff
@@ -10,9 +19,29 @@ export async function getUser() {
 }
 
 export async function getFolderInfo() {
-  const response = await fetch(
-    "https://bootcamp-api.codeit.kr/api/sample/folder"
-  );
+  const response = await fetch(`${BASE_URL}/sample/folder`);
+  if (!response.ok) {
+    throw new Error("폴더 정보를 불러오는데 실패했습니다");
+  }
+  const body = await response.json();
+  return body;
+}
+
+export async function getFolderList({ userId }) {
+  const response = await fetch(`${BASE_URL}/users/${userId}/folders`);
+  if (!response.ok) {
+    throw new Error("폴더 정보를 불러오는데 실패했습니다");
+  }
+  const body = await response.json();
+  return body;
+}
+
+export async function getLinks({ userId, folderId }) {
+  let query = `?folderId=${folderId}`;
+  if (!folderId) {
+    query = "";
+  }
```

Comment:
P3:
재할당이 되는 let보다 가능하면 const를 사용하시면 좋겠습니다.

```suggestion
  const query = folderId ? `?folderId=${folderId}` : '';
```